modules/fedimint-wallet-common/src/config.rs

```diff
@@ -52,6 +57,11 @@ pub struct WalletGenParamsConsensus {
     pub finality_delay: u32,
     /// See [`WalletConfigConsensus::client_default_bitcoin_rpc`].
     pub client_default_bitcoin_rpc: BitcoinRpcConfig,
+    /// Fees to be charged for deposits and withdraws _by the federation_ in
+    /// addition to any on-chain fees.
+    ///
+    /// Deposit fees in particular are a protection against dust attacks.
+    pub fee_consensus: FeeConsensus,
```

Comment:
Out of scope, but it may be useful to dynamically update fees. It's an edge case, but if a federation observes the threshold is too low for a given fee environment it would be useful to update without restarting the federation with a new config.

Reply:
Tracked as part of #4751. I hope we can get generalized dynamic config somehow, but worst case we could special-case certain config parts. @joschisan is researching this problem, the general case is quite complex unfortunately.

Reply:
Thanks for the link! Sounds good, certainly not in scope for this PR.